Source/Module/SearchRepository/SearchRepoScreen.swift

```diff
@@ -10,31 +10,23 @@ import SwiftUI
 
 struct SearchRepoScreen: View {
 
+    @ObservedObject var viewModel: SearchRepoViewModel
     @Binding var showSearchScreen: Bool
+    @ObservedObject var notifiedReposObservable = NotifiedRepositoryObservable()
 
-    @State private var searchText = "" {
-        didSet {
-            print(searchText)
-        }
-    }
     @State private var isShowing = true
 
-    @ObservedObject var notifiedReposObservable = NotifiedRepositoryObservable()
-
     var body: some View {
         VStack(alignment: .leading) {
-            SearchBar(text: $searchText, placeholder: "Search repositories to notify")
+            SearchBar(text: $viewModel.searchText, placeholder: "Search repositories to notify")
                 .border(.darkBorder, width: 1.0, radius: 8.0)
                 .frame(maxWidth: .infinity)
                 .paddingLeft8()
 
             ScrollView(.vertical) {
                 LazyVStack(alignment: .leading) {
-                    ForEach(Array(RepositoryCellViewModel.dummy.enumerated()), id: \.offset) { index, item in
+                    ForEach(viewModel.filteredItems) { item in
                         RepositoryCell(viewModel: item, isNotifying: notifiedReposObservable.getIsNotifying(item))
-                            .background(
-                                index % 2 == 0 ? Color.white : Color.gray.opacity(0.1)
-                            )
```

Comment:
Is it possible to keep these line with
```
ForEach(viewModel.filteredItems.enumerated()) { index, item in
```

Reply:
I see, there's a bug with catalyst when search bar is active and scrolling.

Reply:
This block seems to work fine
```

            ScrollView(.vertical) {
                LazyVStack(alignment: .leading) {
                    ForEach(Array(zip(viewModel.filteredItems.indices, viewModel.filteredItems)), id: \.0) { index, item in
                        RepositoryCell(viewModel: item, isNotifying: notifiedReposObservable.getIsNotifying(item))
                            .background(
                                index % 2 == 0 ? Color.white : Color.gray.opacity(0.1)
                            )
                    }
                }
                .cornerRadius(8.0)
            }
```

Currently I found this bug where the scrollview stop working, I'm looking at it currently but not sure why. If you are available could you help me look. It's easier to replicate if you search multiple times.

https://user-images.githubusercontent.com/6356137/134111364-4731185b-c103-4041-9497-ee327aaa3871.mp4

Reply:
@blyscuit I faced it too, but not much time to search and solve it tbh 🙈

Reply:
> This block seems to work fine
> ```
> 
>             ScrollView(.vertical) {
>                 LazyVStack(alignment: .leading) {
>                     ForEach(Array(zip(viewModel.filteredItems.indices, viewModel.filteredItems)), id: \.0) { index, item in
>                         RepositoryCell(viewModel: item, isNotifying: notifiedReposObservable.getIsNotifying(item))
>                             .background(
>                                 index % 2 == 0 ? Color.white : Color.gray.opacity(0.1)
>                             )
>                     }
>                 }
>                 .cornerRadius(8.0)
>             }
> ```

really nice suggestion, it works perfectly, thank you <3